### DISCUSSION TITLE
W3mQrCode  component is not displaying the Qr code

### DISCUSSION THREAD

**Discussion by abdul-kabugu** [WalletConnect/walletconnect-monorepo#3968] [Developer Support]
### I'm working on polkadot project  and  I'm using  wc  for  connecting mobile wallet 
  i used W3mQrCode to  generate  custom qr  code   but  when i  try to  render the component it shows  nothing  

**Code  included  uri test **  

`  <div className='bg-white'>
         <W3mQrCode size={100}  imageUrl="url/to/image" uri={`wc:23d35f3f31ab2efde7099de9e4b11aa30c9202b8d629576a3e3e835b313ffeec@2?relay-protocol=irn&symKey=020c54634e607ed2022abe990bdcf1156ddf9ee075c9f1e30f9f673552d1d81c`} className='bg-red-600 text-white' />
         </div>`

**what i see in browser** 

![image](https://github.com/WalletConnect/walletconnect-monorepo/assets/68326324/151d1160-c23a-4bc3-a7a7-c751d2d78094)

**Comment by glitch-txs** [comment]
hi, we're not maintaining Web3Modal v2 anymore, please upgrade to v3 🙏
